web-app/components/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/jsx-props-no-spreading */
+const navigation = {
+  main: [
+    { name: 'Discord', href: 'https://discord.gg/k4pxafjMWA' },
```

Comment:
Please move Discord and YouTube to the "social" section and use icons for them that match the styles of the others. I've never seen this inline SVG approach. We might want to do something simpler, like use https://fontawesome.com/search?q=discord&s=solid%2Cbrands (if you can find a free version that we're allowed to use) or a similar library / tool.

Reply:
E.g. maybe https://ionic.io/ionicons or https://mdbootstrap.com/docs/standard/extended/social-media/ I'm forgetting other possibilities.

Reply:
I used Font Awesome and move all socials to one place and move 'Jobs' to navbar